### PR TITLE
Ui Update

### PR DESCRIPTION
**Pull request by grudger007**
Changed static data to links

### CODE REVIEW

File: imageprocessor/templates/base.html [hunk 1]
```diff
@@ -21,7 +20,7 @@
   <header>
 		  <nav class="navbar navbar-expand-lg navbar-light bg-light" style="height: 60px;">
 		  <a class="navbar-brand" href="/">
-			<img src="{% static 'img/logo.png' %}" width="100" height="100" alt="Photo Tagger">
+			<img src="https://image.ibb.co/h3yDUf/logo.png" width="100" height="100" alt="Photo Tagger">
```
Comment (kmcdon10) [inline imageprocessor/templates/base.html]: What does this look like if the image is no longer available?

Comment (grudger007) [reply imageprocessor/templates/base.html]: If image is not available then it will replace it with alt text, which in this case is "Photo Tagger".

Comment (kmcdon10) [reply imageprocessor/templates/base.html]: Have you tried it out? I don't think that's what happens. I'd rather see the images replaced with an onerror attribute, with something like this

**Review by kmcdon10 — APPROVED** [review]

File: imageprocessor/templates/base.html (hunk 1 above)
Comment (grudger007) [reply imageprocessor/templates/base.html]: added onerror attribute which hides the image if not found.

**Pull request merged**